PIM/API/06_ManageVariantProducts.md

```diff
@@ -0,0 +1,115 @@
+# Manage variant products
+
+You can create a single or multiple variant products to an existing master or a master-to-be single product via API addressing the endpoint */Actindo.Modules.Actindo.PIM.Products.createVariants*. 
+
+To edit and delete *PIM* variant products via API, see [Manage products](./05_ManageProducts.md). 
+
+> [Caution] In the latest version of the *PIM* and *DataHub* modules, it is possible to modify the attribute key. Bear in mind, though, that attribute keys are used for API access. Changing the attribute key, which is strongly discouraged, has an impact on the API naming for all affected entities where this attribute is included. That means that all external API consumers, workflows, webhooks, or other custom code need to be adjusted accordingly in case any of them address this attribute.
```

Comment:
```suggestion
> [Caution] In the latest version of the *PIM* and *DataHub* modules, it is possible to modify the attribute key. Bear in mind, though, that attribute keys are used for API access. Changing the attribute key, which is strongly discouraged, has an impact on the API naming for all affected entities where this attribute is included. This means that all external API consumers, workflows, webhooks, or other custom code need to be adjusted accordingly in case any of them address this attribute.
```